bedrock/products/templatetags/misc.py

```diff
@@ -281,7 +283,7 @@ def vpn_product_referral_link(
 
 # RELAY ================================================================
 
-RELAY_12_MONTH_PLAN = "yearly"
+RELAY_12_MONTH_PLAN = "monthly_when_yearly"
```

Comment:
I would lean on constants a bit more throughout this code. E.g. set:
```python
RELAY_MONTH_PLAN = "monthly"
RELAY_12_MONTH_PLAN = "monthly_when_yearly"
```

And below on line 306 you can do:
```python
plan = RELAY_MONTH_PLAN if plan == RELAY_MONTH_PLAN else RELAY_12_MONTH_PLAN
```

This also helps in case of string typos -- if the constant is mistyped a linter will catch it.

Reply:
Should I change the jinja templates to reference the constants too?

Reply:
If it makes sense to do so. But mostly I was concerned with the Python strings and taking advantage of the linting in this file.

Reply:
I'll leave it as is then. If it's good to go can I get an approved review please?